provisioner/ibm-s3fs-provisioner.go

```diff
@@ -137,7 +136,7 @@ var _ controller.Provisioner = &IBMS3fsProvisioner{}
 var writeFile = ioutil.WriteFile
 
 func UnixConnect(addr string, t time.Duration) (net.Conn, error) {
-	unix_addr, err := net.ResolveUnixAddr("unix", addr)
+	unix_addr, err := net.ResolveUnixAddr("unix", addr) // nolint:ineffassign
```

Comment:
Maybe replacing `unix_addr, err := net.ResolveUnixAddr("unix", addr)` ---> `unix_addr, _ := net.ResolveUnixAddr("unix", addr)` should fix this as we are not utilizing this `err` futher

Also is there any possibility that `net.ResolveUnixAddr` might fail? If yes, then I think we  may need to return `err` after this call itself.

Reply:
The error we are handling in next line where we pass unix_addr `conn, err := net.DialUnix("unix", nil, unix_addr)` , so for 139 line I will ignore error.